waspc/data/Generator/templates/react-app/src/components/Loader.module.css

```diff
@@ -0,0 +1,37 @@
+.loader {
+  color: var(--loader-color, #1a202c);
```

Comment:
Wrote it like this to enable users to adjust the loader color with CSS vars. It's really documented but we can tell users that if they ask.

Reply:
So this is still local CSS, is it? Not global, right? Because of the ".module.css"? WHo is enabling this, Vite, or are we using something special?

Reply:
Vite supports CSS modules out of the box: https://vitejs.dev/config/shared-options.html#css-modules